html/themes/custom/common_design_subtheme/common_design_subtheme.libraries.yml

```diff
@@ -132,6 +132,8 @@ rw-form:
     theme:
       components/rw-form/rw-form.css: {}
       components/rw-form/rw-form--widget.css: {}
+  dependencies:
+    - common_design/cd-form
```

Comment:
I've added this as a dependency because I noticed it did not load on the revision confirmation pages, despite `rw-form` component loading. I want to run VRT to confirm there are no negative affects.